src/elements/play-preview.ts

```diff
@@ -125,7 +125,7 @@ export class PlayPreview extends LitElement {
     'devvit-installation': {values: ['123']},
     'devvit-subreddit': {values: ['t5_123']},
     'devvit-user': {values: ['t2_123']},
-    'devvit-user-agent': {values: ['play']}
+    'devvit-user-agent': {values: ['Reddit;Play;not-provided']}
```

Comment:
I'm unsure what the overlap is with an HTTP user agent and metadata. most metadata uses multiple items or a single CSV to represent multiple values.

Reply:
This is the structure that iOS/Android gave and I wanted to make it consistent across all surfaces 🤷